components/overledger/actions/sign-a-transaction/sign-a-transaction.mjs

```diff
@@ -0,0 +1,77 @@
+import overledger from "../../overledger.app.mjs";
+import { UNIT_OPTIONS } from "../../common/constants.mjs";
+
+export default {
+  key: "overledger-sign-a-transaction",
+  name: "Sign a transaction",
+  description: "Sign a transaction using Overledger",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    overledger,
+    environment: {
+      propDefinition: [
+        overledger,
+        "environment",
+      ],
+    },
+    locationTechnology: {
+      type: "string",
+      label: "Location Technology",
+      description: "The blockchain technology used for this transaction, e.g., ethereum, substrate - required in order to set the dltfee",
+      //previous prep step output - reuired in this step in order to set the dltfee unit
+      default: ({ steps }) => steps.prepare_smart_contract_transaction?.locationTechnology || "ethereum",
```

Comment:
It doesn't work
```suggestion
```

Reply:
Hi @luancazarine . Do you suggest I just remove this default value? Has it been tested with the Prep smart contract  step before hand?

Reply:
done